frontend/src/views/SecretOverviewPage/SecretOverviewPage.tsx

```diff
@@ -110,6 +110,10 @@ export const SecretOverviewPage = () => {
   const userAvailableEnvs = currentWorkspace?.environments || [];
   const [visibleEnvs, setVisisbleEnvs] = useState(userAvailableEnvs);
 
+  useEffect(() => {
+    setVisisbleEnvs(userAvailableEnvs);
```

Comment:
`setVisisbleEnvs` is also a typo. I know it's not done by you, but if we could fix this in this PR as well then that could be great!